C-SharpBasic/CSharp-Assignment-2/ReadFile.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace C_Sharp
+{
+    internal class ReadFile
+    {
+            String check;
+            public void Read()
+            {
+                try
+                {
+
+                    StreamReader sr = new StreamReader(@"C:\Users\Ajinkya.Sonwane\source\repos\C-Sharp\CSharp-Assignment-2\C# Assignments – II.txt");
+
+                    check = sr.ReadLine();
```

Comment:
what does check mean?

Reply:
It contains text data from file in string format. name is change is stringContainer.